tox.ini

```diff
@@ -12,7 +18,9 @@ python =
 [testenv]
 commands = python run_tests.py
 deps =
-    flake8 >= 5.0.0
+    flake8-v5: flake8 >= 5.0.0, < 6.0.0
+    flake8-v6: flake8 >= 6.0.0, < 7.0.0
+    flake8-next: flake8
```

Comment:
I think `-latest` is a more accurate name than `-next`, given what this matches.